core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/AbstractShaclTest.java

```diff
@@ -439,7 +439,7 @@ private static void testValidationReport(String dataPath, Model validationReport
 			Model validationReportExpected = Rio.parse(resourceAsStream, "", RDFFormat.TURTLE);
 
 			if (!isIsomorphic(validationReportActual, validationReportExpected)) {
-				writeActualModelToExpectedModelForDevPurposes(dataPath, validationReportActual);
+//				writeActualModelToExpectedModelForDevPurposes(dataPath, validationReportActual);
```

Comment:
As it says, this was for dev purposes and should have been commented out already. Fixed now though!